docs/pages/router/migrate/from-react-navigation.mdx

```diff
@@ -108,7 +108,7 @@ function App() {
       /* @info Delete the linking configuration, this is managed by Expo Router. */
       linking={
         {
-          /* @hidden ... */ /* @end */
+          // ...
```

Comment:
```suggestion
          /* @hide ... */ /* @end */
```

We often `@hide` annotation to describe descriptive code length in a snippet. There was a typo with the previous one (`@hidden` could've been `@hide`).

Reply:
I did this but then the copy action was brining over a `...` which is invalid JS

Reply:
Interesting. I will look into it. Thank you for the input.